packages/app/src/app/pages/common/Stats/index.tsx

```diff
@@ -9,7 +9,11 @@ import Stat from './Stat';
 import { Stats } from './elements';
 import { Sandbox } from '@codesandbox/common/lib/types';
 
-function StatsComponent({ sandbox }: { sandbox: Sandbox }) {
+interface Props {
+  sandbox: Sandbox;
+}
+
+const StatsComponent = ({ sandbox }: Props) => {
   return (
```

Comment:
@Saeris This can be an implicit  return

```suggestion
const StatsComponent = ({ sandbox }: Props) => (
```